### PR TITLE
dev-python/docopt: enable pypy build targets

### PR DESCRIPTION
**Pull request by atom-symbol**
Notes:
- Tested via app-text/grip
- There is no bugs.gentoo.org issue related to this pull request

### CODE REVIEW

File: dev-python/docopt/docopt-0.6.2-r1.ebuild [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 EAPI=5
 
-PYTHON_COMPAT=( python2_7 python3_{3,4,5} )
+PYTHON_COMPAT=( python2_7 python3_{3,4,5} pypy pypy3 )
```
Comment (rindeal) [inline dev-python/docopt/docopt-0.6.2-r1.ebuild]: could become `python{2_7,3_{3,4,5}} pypy{,3}`

Comment (atom-symbol) [inline dev-python/docopt/docopt-0.6.2-r1.ebuild]: `grep -R "pypy pypy3" /usr/portage/dev-python/` shows that the right way of changing "pypy pypy3" into "pypy{,3}" is to write a script to do the task.

**Comment by monsieurp** [comment]
@gentoo/python

**Comment by monsieurp** [comment]
Hi and thanks for the PR. Can you provide a build log to make sure it does compile with pypy? Thanks!

**Comment by atom-symbol** [comment]
@monsieurp The pull request contains the test "Tested via app-text/grip". I won't provide the build log.

**Comment by monsieurp** [comment]
I don't want to be finicky and I'm usually not that kind of person but please do explain to me how does "tested via blabla" make your statement valid? We need a proof to make sure it does work.

**Comment by atom-symbol** [comment]
@monsieurp I cannot make [the decision whether to implicitly trust the pull request submitted by me] for you. Merge it, or close it without merging.

Hint: If you value efficient use of time, you will arrive at the decision quickly.

Thanks.

**Comment by monsieurp** [comment]
```Hint: If you value efficient use of time, you will arrive at the decision quickly.```

Listen, PRs are a tool we use to streamline the process of merging contributions. The main platform for accepting contributions is and will remain for the foreseeable future https://bugs.gentoo.org. Never ever forget that. If you filed a bug report with the same diff and wanted us to merge it, we would ask you to prove that it does work and provide a build log as a proof. Full stop. *Just* because you took the time to file a Pull Request does NOT guarantee your contribution will end up in the Gentoo tree. There's no guarantee whatsoever. The sooner you understand that, the better for everybody. Thank you.

**Comment by atom-symbol** [comment]
@monsieurp Well done.

**Comment by idella** [comment]
@atom-symbol not sure what you're playing at here but I have some clues. Are you familiar with the term intransigence?  The emoticons appear to be pure defiance.
1. I would emerge pypy and cross test
2  I would then emerge pypy3 and cross test
3. pypy takes around 160 minutes and I purged it to make things easier on the assumption no-one cares about pypy
4. Frankly I do not see the need for a build log.  the output from yr own testrun free of error ought suffice.
5. Users contribute and their contribution is reviewed by members of the community. You are under the illusion you can dictate terms to us, you have everything the wrong way around.

**Comment by titanofold** [comment]
@atom-symbol, thank you for the work you've put forth in this PR. We are declining to merge at this time for the following reasons:

- Inappropriate response to a constructive suggestion
- Refusal to provide requested data
- Rudeness to a fellow GitHub member
- No mention of benefit of supporting another Python version

If you wish to discuss this matter, you may contact us via email at python@gentoo.org or via IRC at #gentoo-python on Freenode.

**Comment by idella** [comment]
done

**Pull request closed without merging**